app/utils/print/BusinessSettlorPrintHelper.scala

```diff
@@ -32,6 +32,9 @@ class BusinessSettlorPrintHelper @Inject()(answerRowConverter: AnswerRowConverte
         bound.stringQuestion(NamePage, "businessSettlor.name", Some(controllers.business.routes.NameController.onPageLoad(NormalMode).url)),
         bound.yesNoQuestion(UtrYesNoPage, "businessSettlor.utrYesNo", Some(controllers.business.routes.UtrYesNoController.onPageLoad(NormalMode).url)),
         bound.stringQuestion(UtrPage, "businessSettlor.utr", Some(controllers.business.routes.UtrController.onPageLoad(NormalMode).url)),
+        bound.yesNoQuestion(CountryOfResidenceYesNoPage, "businessSettlor.countryOfResidenceYesNo", Some(controllers.business.routes.CountryOfResidenceYesNoController.onPageLoad(NormalMode).url)),
```

Comment:
Will leave it up to you whether you want to change it or not but with the print helpers in beneficiaries I've been changing it so we don't repeat what are effectively the same lists. For reference: https://github.com/hmrc/maintain-beneficiaries-frontend/blob/master/app/utils/print/CharityBeneficiaryPrintHelper.scala

Reply:
Yep, that makes sense. I'll do that